index.html

```diff
@@ -67,5 +67,130 @@ <h5 class="small-text">I can help you build a product , feature or website Look
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
+           </p>
+           <ul class="tags">
+            <li class="tag-item">CSS</li>
+            <li class="tag-item">HTML</li>
+            <li class="tag-item">Bootstrap</li>
+            <li class="tag-item">Ruby</li>
+           </ul>
+           <button type="button">See Project</button>
+        </div>
+        <div class="works-grid">
+           <div class="work-item">
+            <div class="work-item-content">
+                <h3>Multi-Post Stories</h3>
+                <p>
+                    A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
+                </p>
+                <ul class="tags">
+                    <li class="tag-item">CSS</li>
+                    <li class="tag-item">HTML</li>
+                    <li class="tag-item">Bootstrap</li>
+                    <li class="tag-item">Ruby</li>
+                </ul>
+            </div>
+            <button type="button">See Project</button>
+           </div> 
+           <div class="work-item">
+            <div class="work-item-content">
+                <h3>Multi-Post Stories</h3>
+                <p>
+                    A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
+                </p>
+                <ul class="tags">
+                    <li class="tag-item">CSS</li>
+                    <li class="tag-item">HTML</li>
+                    <li class="tag-item">Bootstrap</li>
+                    <li class="tag-item">Ruby</li>
+                </ul>
+            </div>
+            <button type="button">See Project</button>
+        </div>
+        <div class="work-item">
+            <div class="work-item-content">
+                <h3>Multi-Post Stories</h3>
+                <p>
+                    A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
+                </p>
+                <ul class="tags">
+                    <li class="tag-item">CSS</li>
+                    <li class="tag-item">HTML</li>
+                    <li class="tag-item">Bootstrap</li>
+                    <li class="tag-item">Ruby</li>
+                </ul>
+            </div>
+            <button type="button">See Project</button>
+            </div>
+            </div>
+    </section>
+
+    <section class="about-page">
+      <h1 class="large-intro">
+      About Me  
+      </h1>  
```

Comment:
- [ ] Kindly use the correct font sixes for this section as provided by the Figma design.

**This is how the section looks like:**

![image](https://user-images.githubusercontent.com/86318284/185756887-a67bd3f1-915f-4b80-ad72-a525eddcf1da.png)

**This is the desired Figma design:**

![image](https://user-images.githubusercontent.com/86318284/185756903-1808d911-9cec-4af0-b31a-ba0c777b5cd7.png)

- [ ] Kindly note that header 1 can only be one for a given HTML webpage. Kindly consider using either header two instead.